src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -588,6 +589,15 @@ export const examManagementRoute: Routes = [
         },
         canActivate: [UserRouteAccessService],
     },
+    {
+        path: ':examId/exercise-groups/:exerciseGroupId/programming-exercises/:exerciseId/grading/:tab',
+        component: ProgrammingExerciseConfigureGradingComponent,
+        data: {
+            authorities: [Authority.EDITOR, Authority.INSTRUCTOR, Authority.ADMIN],
```

Comment:
Grading button is enabled for Instructor and higher roles.(I just look at the template file.) I think we should organize authorizations according to that.

Reply:
On a second thought, as editors as there to create exercises, we should probably also allow those to access the grading page, or not?

Reply:
I copied those from the other route, apparently the button in the template should use isAtLeastEditor.

Reply:
I checked back with Stephan and grading should be accessible for editors, so I fixed the auth check in the template in https://github.com/ls1intum/Artemis/pull/3748/commits/6cfcfea477e34787acf7c8ac4b07700a14e5c409.